lib/rucio/client/subscriptionclient.py

```diff
@@ -135,8 +137,15 @@ def update_subscription(self, name, account=None, filter_=None, replication_rule
         :type priority: Integer
         :raises: exception.NotFound if subscription is not found
         """
+        from rucio.client import Client
```

Comment:
Maybe better to just include the client that actually has whoami(), and not all of them?